FinalProject/StateController.cpp

```diff
@@ -97,15 +96,22 @@ DWORD StateController::handleProtocolWriteEvents() {
 				else if (indexOfSignaledEvent == 4) {
 					setState(RTS);
 					releaseTX = !releaseTX;
+					break;
```

Comment:
Don't think we need a break here

Reply:
Right, we can remove it. We'll need to revisit this block of code to handle resending correctly.